### PR TITLE
DBZ-3270 DBZ-3384 Add dynamic logging information

### PR DESCRIPTION
**Pull request by mpermar**

### CODE REVIEW

File: documentation/modules/ROOT/pages/operations/logging.adoc [hunk 1]
```diff
@@ -126,7 +126,16 @@ In general, {prodname} connectors send their log messages to loggers with names
 {prodname} uses packages to organize code with similar or related functions.
 This means that you can control all of the log messages for a specific class or for all of the classes within or under a specific package.
 
-.Procedure
+.Dinamically changing logging level
```
Comment (gunnarmorling) [inline documentation/modules/ROOT/pages/operations/logging.adoc:129]:
```suggestion
.Dynamically changing logging level
```

File: documentation/modules/ROOT/pages/operations/logging.adoc [hunk 2]
```diff
@@ -126,7 +126,16 @@ In general, {prodname} connectors send their log messages to loggers with names
...
+
+Kafka Connect allows to change logging level dinamically without restarting your server. For doing that, you can send a PUT request to the admin/loggers endpoint providing the package name for which you want to change log level. For example, if by any reason you wanted to get debug information for the Oracle connector you could send this request to your Debezium server:
```
Comment (gunnarmorling) [inline documentation/modules/ROOT/pages/operations/logging.adoc:131]:
```suggestion
Kafka Connect allows to change logging level dynamically without restarting your server. For doing that, you can send a PUT request to the admin/loggers endpoint providing the package name for which you want to change log level. For example, if by any reason you wanted to get debug information for the {prodname} Oracle connector you could send this request to Kafka Connect:
```

File: documentation/modules/ROOT/pages/operations/logging.adoc [hunk 3]
```diff
@@ -126,7 +126,16 @@ In general, {prodname} connectors send their log messages to loggers with names
 {prodname} uses packages to organize code with similar or related functions.
 This means that you can control all of the log messages for a specific class or for all of the classes within or under a specific package.
 
-.Procedure
```
Comment (gunnarmorling) [inline documentation/modules/ROOT/pages/operations/logging.adoc:129]: Could you add this back, it's an established convention in the docs.

**Review by gunnarmorling — COMMENTED** [review]
Thanks a lot! Some small comments inline. Could you also attribute this to DBZ-3270, which was logged for this improvement before. Thx!

**Comment by gunnarmorling** [comment]
@roldanbob, you're lucky, @mpermar from the community has done most of the work for DBZ-3270 :) After the comments above have been addressed, could you please check out this PR and apply further changes as you see fit? Thanks!

**Comment by mpermar** [comment]
Thanks for the fixes @gunnarmorling . Updated the PR.

**Comment by roldanbob** [comment]
> @roldanbob, you're lucky, @mpermar from the community has done most of the work for DBZ-3270 :) After the comments above have been addressed, could you please check out this PR and apply further changes as you see fit? Thanks!

Thanks for your help @mpermar!

@gunnarmorling I reworked the content to comply with formatting requirements. I also updated the API example to target MySQL vs. Oracle, since the latter is not yet supported downstream.

**Comment by gunnarmorling** [comment]
Rebased and applied. Thanks a lot, @mpermar and @roldanbob!

**Pull request closed without merging**